src/cco-extensions/ModalRelationOntology.ttl

```diff
@@ -2268,7 +2268,7 @@ mro:is_permitted_by rdf:type owl:ObjectProperty ;
                     owl:inverseOf mro:permits ;
                     rdfs:domain obo:BFO_0000015 ;
                     rdfs:range cco:ActionRegulation ;
-                    cco:definition "y is_permitted_by x at t iff: x is an instance of Action Regulation at time t, and y is an instance of Act at time t, and x prescribes that some agent may be agent in y."@en ;
+                    cco:definition "y is_permitted_by x at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that some independent continuant (but not spatial regiom) may participate in y."@en ;
                     cco:is_curated_in_ontology <http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ModalRelationOntology> ;
```

Comment:
@cameronmore 'region' has a mispelling. 

Also, what do you think of: "y is_permitted_by x at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that y may occur." 

If acceptable, this phrasing would apply to multiple object properties in this PR.